### ISSUE TITLE
Suggestion - Google news

### ISSUE THREAD

**Issue by xiaoming-w** [deanishe/alfred-searchio#59]
Hi 

Thanks a lot for the hard work on this lovely alfred workflow, I use it almost everyday.

I was wondering if we could have Google news built in, that would be an awesome plus.

Thank you!

**Comment by deanishe** [comment]
> I was wondering if we could have Google news built in

I don't think so. It doesn't support OpenSearch, and there's no autosuggestion API that I can find.

**Comment by xiaoming-w** [comment]
> > I was wondering if we could have Google news built in
> 
> I don't think so. It doesn't support OpenSearch, and there's no autosuggestion API that I can find.

Thanks for the reply. just for your information, Google has 2 news portal:
one is the news tab in google.com, like what's shown below
![image](https://user-images.githubusercontent.com/43328230/86633196-5e890f00-bf9e-11ea-8daa-b0e98a7b461f.png)

another one is an independent portal:
![image](https://user-images.githubusercontent.com/43328230/86633273-75c7fc80-bf9e-11ea-9019-72bbfe8f7fc5.png)

both of these two are called "google news", but work slightly different.

I was wondering if you've checked both to find the autosuggestion API

Thanks

**Comment by deanishe** [comment]
> I was wondering if you've checked both to find the autosuggestion API

No, I didn't. I only checked Google News, not the News tab of regular search, though after a cursory look, I see no indication of any news-specific autosuggestion API there, either.

To be honest, I'm not going to spend more than a few minutes trying to find an appropriate API for a given website, unless it's one that I personally care about.

If *you* do the legwork of finding the API for the site you're interested in, there's a much better chance that it'll get added.

**Comment by xiaoming-w** [comment]
> > I was wondering if you've checked both to find the autosuggestion API
> 
> No, I didn't. I only checked Google News, not the News tab of regular search, though after a cursory look, I see no indication of any news-specific autosuggestion API there, either.
> 
> To be honest, I'm not going to spend more than a few minutes trying to find an appropriate API for a given website, unless it's one that I personally care about.
> 
> If _you_ do the legwork of finding the API for the site you're interested in, there's a much better chance that it'll get added.

Yeah, sure, I'll look into it and see if I can find the API, will let you know.

Thanks
